translations/messages+intl-icu.fr.xlf

```diff
@@ -434,6 +434,63 @@
                 <source>paginator.next</source>
                 <target>Suivant</target>
             </trans-unit>
+
+            <trans-unit id="action.forgot_password">
+                <source>action.forgot_password</source>
+                <target>Mot de passe oublié</target>
+            </trans-unit>
+            <trans-unit id="reset_password.title">
+                <source>reset_password.title</source>
+                <target>Réinitialiser votre mot de passe</target>
+            </trans-unit>
+
+            <trans-unit id="reset_password.check_email.p1">
+                <source>reset_password.check_email.p1</source>
+                <target>Un e-mail contenant un lien sur lequel vous pouvez cliquer pour réinitialiser votre mot de passe a été envoyé. Ce lien expirera dans {token} heure (s).</target>
```

Comment:
`heure (s).` =>  `heure(s).`

Reply:
even better would be to the the ICU `plural` rules.